pandas_datareader/edgar.py

```diff
@@ -1,19 +1,23 @@
 import re
+import pandas as pd
 import datetime as dt
 from ftplib import FTP
 import gzip
 
-from pandas import read_csv
-from pandas import DataFrame
-from pandas import to_datetime
 from zipfile import ZipFile
 from pandas.compat import StringIO
-from pandas.api.types import is_number
+from distutils.version import LooseVersion
+from pandas import read_csv, DataFrame, to_datetime
 
 from pandas_datareader.base import _BaseReader
 from pandas_datareader.compat import BytesIO
 from pandas_datareader._utils import RemoteDataError
 
+if LooseVersion(pd.__version__) <= '0.19.2':
```

Comment:
would put this in ``pandas_datareader/compat/__init__.py``

```
if LooseVersion(.....):
       from pandas.core import common as pdapi
else:
      from pandas.api import types as pdapi
```

then you can 

``from pandas_datareader.compat import pdapi``